### PR TITLE
Tweak JS API use

### PR DESCRIPTION
**Pull request by one-more-minute**
Hope you don't mind me making this package even smaller ;) Implementing `Blink.js()` for `Plot`s gives us a bunch of nice interop utilities for free. This will need Blink.jl master for now due to the use of a new eval util as well as the new promise support.

### CODE REVIEW

**Comment by tbreloff** [comment]
serious concisification ;)

File: src/display.jl [hunk 1]
```diff
@@ -78,81 +78,38 @@ end
...
-    _call_js(p, "Plotly.relayout($thediv, $update);")
-end
-
-function addtraces!(p::Plot, traces::AbstractTrace...)
-    thediv = "document.getElementById('$(p.divid)')"
-    update = json(traces)
-    _call_js(p, "Plotly.addTraces($thediv, $update);")
-end
+addtraces!(p::Plot, traces::AbstractTrace...) =
+    @js_ p Plotly.addtraces(this, $traces)
```
Comment (spencerlyon2) [inline src/display.jl]: Typo here. This should be `Plotly.addTraces` with a capital `T`

**Comment by spencerlyon2** [comment]
Awesome changes! Thank you for cleaning this up. It is much much better now.

File: src/display.jl (hunk 1 above)
Comment (one-more-minute) [inline src/display.jl]: Whoops, good catch, will fix that up

**Comment by spencerlyon2** [comment]
Looks great, thanks for the cleanup

**Pull request merged**